### ISSUE TITLE
Upgrade failure

### ISSUE THREAD

**Issue by muzzmac** [jeatheak/Mitsubishi-WF-RAC-Integration#92]
I attempted to update the integration via the Home Assistant GUI. 

When I press the "Update" button, it spins forever and never completes...
![image](https://github.com/jeatheak/Mitsubishi-WF-RAC-Integration/assets/101535858/22e0522e-e61c-4198-b464-f61dd1cd22f5)

![image](https://github.com/jeatheak/Mitsubishi-WF-RAC-Integration/assets/101535858/d51b617c-c97e-4ff0-8741-755bdca11a9f)

I can't seem to see any option to remove the integration to try a new install. The integration worked before I attempted the upgrade but no longer starts post update attempt.

Any problem solving thoughts?

**Comment by jeatheak** [comment]
Which version of home assist core are you running?

**Comment by muzzmac** [comment]
I just managed to resolve the underlying issue. 

The root cause of all of this is my local network is IPv6 and IPv4 and there seems to be an issue with downloading from HACS using IPv6. 

Disabling IPv6 has resolved HACS install issues.

Something to keep in mind. The download button spinning and never loading with a HACS install will likelyt be that.

This is the article I found that helped me resolve this.

https://community.home-assistant.io/t/just-installed-hacs-unable-to-download-anything-due-to-forever-loading/375834
